modules/broker-amqp/src/main/java/io/ballerina/messaging/broker/amqp/codec/AmqpChannel.java

```diff
@@ -418,6 +421,14 @@ public boolean isTransactional () {
         return transaction.isTransactional();
```

Comment:
Do we need the isTransactional method?

Reply:
Updated the method. Removed isTransactional property from BrokerTransaction object